### PR TITLE
Fix channel verification status api.

### PR DESCRIPTION
**Pull request by kameshsr**

### CODE REVIEW

File: resident/resident-service/src/main/java/io/mosip/resident/util/Utilitiy.java [hunk 1]
```diff
@@ -440,4 +449,29 @@ public String getFileName(String eventId, String propertyName){
...
+		if (identityDTO != null) {
+			uin = identityDTO.getUIN();
+			email = identityDTO.getEmail();
+			phone = identityDTO.getPhone();
+		}
+		String idaToken= identityService.getIDAToken(uin);
+		String id;
+		if(email != null && phone !=null && channel.size()==2) {
+			id= email+phone+idaToken;
+		}else if(email != null && channel.get(0).equalsIgnoreCase(EMAIL)) {
```
Comment (loganathan-sekaran) [inline resident/resident-service/src/main/java/io/mosip/resident/util/Utilitiy.java]: Make sure the channel size is one.

File: resident/resident-service/src/main/java/io/mosip/resident/util/Utilitiy.java [hunk 2]
```diff
@@ -440,4 +449,29 @@ public String getFileName(String eventId, String propertyName){
...
+			id= email+idaToken;
+		} else if(phone != null && channel.get(0).equalsIgnoreCase(PHONE)) {
```
Comment (loganathan-sekaran) [inline resident/resident-service/src/main/java/io/mosip/resident/util/Utilitiy.java]: make sure the channel size is one

**Review by loganathan-sekaran — CHANGES_REQUESTED** [review]

File: resident/resident-service/src/main/java/io/mosip/resident/util/Utilitiy.java (hunk 1 above)
Comment (kameshsr) [reply resident/resident-service/src/main/java/io/mosip/resident/util/Utilitiy.java]: done.

File: resident/resident-service/src/main/java/io/mosip/resident/util/Utilitiy.java (hunk 2 above)
Comment (kameshsr) [reply resident/resident-service/src/main/java/io/mosip/resident/util/Utilitiy.java]: done.

**Review by loganathan-sekaran — APPROVED** [review]

**Pull request merged**